### DISCUSSION TITLE
Flash back about entering Unity

### DISCUSSION THREAD

**Discussion by Lihao7788** [sofa-framework/sofa#4282] [Write / Run a simulation]
I import the license file into Unity, run the scene, and immediately flash back what is the reason?

**Comment by epernod** [comment]
Hi,

could you open a discussion into this repository: https://github.com/InfinyTech3D/SofaUnity-Renderer/discussions
to keep the record.

Could you also add the unity log crash to this new discussion so we could solve your issue

Thank you
